src/main/java/uk/gov/hmcts/juror/api/moj/service/expense/JurorExpenseServiceImpl.java

```diff
@@ -260,11 +258,11 @@ PayAttendanceType calculatePayAttendanceType(Appearance appearance) {
     @Override
     @Transactional
     @SuppressWarnings("checkstyle:LineLength")
-    public void setDefaultExpensesForJuror(RequestDefaultExpensesDto dto) {
+    public void setDefaultExpensesForJuror(String jurorNumber, RequestDefaultExpensesDto dto) {
 
         final String owner = SecurityUtil.getActiveOwner();
```

Comment:
This will extract the owner of the user (So if logged in as a satellite it will return the primary)
We then use this owner to get all appearances associated to the court (Primary & Satellite) and update any draft expense with the new defaults)